idaes/models_extra/power_generation/flowsheets/test/test_NGFC.py

```diff
@@ -26,32 +26,41 @@
 Tests to make sure the NGFC example builds and solves correctly.
```

Comment:
Not this line but above, there are two copies of the IDAES header for some reason.

Reply:
I tracked the commit history for a few of the files in this directory, and it appears many files gained extra headers as a result of #371 .